### DISCUSSION TITLE
Neper conjunction with comsol

### DISCUSSION THREAD

**Discussion by MII1187** [neperfepx/neper#562] [Q&A] [answered]
Hi，
I want to import the polycrystalline model generated by the neper software into the comsol for simulation, is there any way to achieve it?
Thank you

**Comment by rquey** [comment, marked as answer]
Not with Neper itself.

**Comment by xubaozi1307** [comment]
Hi !
I encountered the same problem as you, did you solve it?

**Reply by MII1187** [reply]
Yes. Convert tess files to stl files. But this method only works for 3D models.
neper -T -n 10 -id 1 -domain "cube(1000,100,100)" -format stl

**Reply by xubaozi1307** [reply]
Thank you very much for your reply, it has helped me a lot

**Reply by uncleyuan001** [reply]
hello, I generated a tess named n100-id1.stl by using your command 'neper -T -n 10 -id 1 -domain "cube(1000,100,100)" -format stl' , but the stl file can't be open by rhino or other available apps

**Reply by xubaozi1307** [reply]
I‘m sorry that  I don't konw why it didn't work，when I use `-format stl`, the tess would be output as a .stl file which can load in COMSOL, maybe you can try other simple commands or check the neper installation, this is my personal small suggestion, hope it helps you.

**Reply by uncleyuan001** [reply]
thank you! I luckily resolved this issue yesterday by correcting the head and the end of the ASC II file
